lib/components/dashboard/course-list/Semester.tsx

```diff
@@ -260,6 +260,12 @@ const Semester: FC<{
       });
     } else {
       console.log('Failed to add', data.errors);
+      data.errors.forEach((error) => {
+        if(error.status == 400) {
```

Comment:
asking cuz i'm clueless; is it really data.errors and not data.error? that's interesting haha

Reply:
yea, thought it was weird too lol